brightlayer-ui/react-themes
# Updated MUI Tooltip font size to .75 rem

Fixes #8.


#### Changes proposed in this Pull Request:
- Updated MUI Tooltip font size to .75 rem


#### Screenshots:
##### Before
![Screen Shot 2022-01-19 at 3 39 11 PM](https://user-images.githubusercontent.com/13989985/150210032-96f8187d-75cd-4d34-969e-079945a6acd6.png)

##### After
![Screen Shot 2022-01-19 at 3 32 26 PM](https://user-images.githubusercontent.com/13989985/150209772-cd554c92-4ac7-4687-8a99-85b7ac82154b.png)



#### To Test:
- Start showcase demo
- Navigate to http://localhost:3000/material-ui-components/data-display-components
- Observe tooltip font size update

note: due to the way that this is implemented by MUI I was unable to verify this change by inspecting the dom tree. I tested this by changing the tooltip font size to a ridiculous size (2rem) to make sure the changes would take and then verified by comparing the new version to the previous version. It may be a good idea to drop a 12px static font on the screen next to the tooltip to verify that it is indeed 12px now.
